bin/all_sky_search/pycbc_template_kde_max

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import numpy, h5py, argparse, logging
+from pycbc import init_logging
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--kde-files', nargs='+', required=True,
+                    help='HDF files with KDE values')
+parser.add_argument('--output-file', required=True, help='Name of output HDF file')
+parser.add_argument('--min-ratio', type=float,
+                    help='Minimum ratio for template_kde relative to the maximum')
+parser.add_argument('--verbose', action='store_true')
+args = parser.parse_args()
+init_logging(verbose=args.verbose, format='%(asctime)s %(message)s')
+
+
+temp_files = [h5py.File(file, 'r') for file in args.template_files]
```

Comment:
I think these lines will fail as the option is now kde_files.  Also better to call the list eg `input_kdes` (this means changes throughout the code). 
Also since 'num_files' is not used beyond this one check, you should not define a new variable (just use `if len(input_kdes) < 2`)